composer.json

```diff
@@ -10,7 +10,7 @@
     ],
     "autoload":{
         "psr-4":{
-            "LCache\\": "src"
+            "LCache\\": ["src/", "tests/"]
```

Comment:
Does this need to go in `autoload`, or could you put it in `autoload-dev`?  I prefer to put autoloaded classes used by the tests in `tests/src`, to differentiate them from files that are loaded by phpunit because they are test files.  See the existing entry in `autoload-dev`. No one is using the LCache\TestUtils namespace; I just put it in as an example. This could just be `LCache`. You could make the directory `tests` instead of `tests/src` too. But I'd recommend keeping it out of `autoload`.